frontend/src/styles.css

```diff
@@ -13,6 +13,11 @@ html,
 body {
   height: 100%;
 }
+
+html {
+  overflow-y: scroll;
+}
+
```

Comment:
We should not have to define this, right? I mean: scroll on overflow is the default.

Reply:
The default is `visible`, not `scroll`.

Reply:
ok